ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/aaa-accounting.j2

```diff
@@ -1,22 +1,26 @@
-{% if aaa_accounting is defined and aaa_accounting is not none %}
+{% if aaa_accounting is arista.avd.defined %}
 
 ## AAA Accounting
 
 ### AAA Accounting Summary
 
 | Type | Sub-type | Record | Accounting Stores | Logging |
 | ---- | -------- | ------ |------------------ | ------- |
-{%     if aaa_accounting.exec.default is defined and aaa_accounting.exec.default is not none %}
-{%        if aaa_accounting.exec.default.group is defined and aaa_accounting.exec.default.group is not none %}
-| Exec | - | {{ aaa_accounting.exec.default.type }} | {{ aaa_accounting.exec.default.group }} | - |
-{%        endif %}
+{%     if aaa_accounting.exec.console.group is arista.avd.defined %}
+| Exec - Console | - | {{ aaa_accounting.exec.console.type }} | {{ aaa_accounting.exec.console.group }} | - |
 {%     endif %}
-{%     if aaa_accounting.commands is defined and aaa_accounting.commands is not none %}
-{%        if aaa_accounting.commands.commands_default is defined and aaa_accounting.commands.commands_default is not none %}
-{%            for command_default in aaa_accounting.commands.commands_default %}
-| Commands | {{ command_default.commands }} | {{ command_default.type }} | {% if command_default.group is defined and command_default.group is not none %}{{ command_default.group }} {% else %} - {% endif %} | {% if command_default.logging is defined and command_default.logging == true %}{{ true }} {% else %} false {% endif %} |
-{%            endfor %}
-{%        endif %}
+{%     if aaa_accounting.commands.commands_console is arista.avd.defined %}
+{%         for command_console in aaa_accounting.commands.commands_console %}
+| Commands - Console | {{ command_console.commands }} | {{ command_console.type }} | {% if command_console.group is arista.avd.defined %}{{ command_console.group }} {% else %} - {% endif %} | {% if command_console.logging is arista.avd.defined(true) %}{{ true }} {% else %} false {% endif %} |
```

Comment:
The inline `if` makes it very hard to read. I suggest to move the logic up above the output line and set some temporary variables to use in the output.
Same below